### PR TITLE
quality processes, new tests cases

### PR DESCRIPTION
**Pull request by tierninho**
Add read-me file for the Quality processes regarding Desktop. Some new test cases to be added to the manifest as well.

### CODE REVIEW

**Comment by iAmWillShepherd** [comment]
@tierninho is this ready for review?

**Comment by niik** [comment]
> @iAmWillShepherd it's ongoing as I am always adding new test cases, so not yet. Will submit for review when 1.1 drops.

@tierninho Now that 1.1 has landed, could we sort out the conflicts and get this ready for review?

File: docs/process/testing.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ### Download Desktop
   - [ ] User can download latest (Mac & Windows) Desktop from https://desktop.github.com/    
     - [ ] Mac: https://central.github.com/deployments/desktop/desktop/latest/darwin
-      - [ ] Homebrew package manager: $ brew cask install github-desktop
+      - [ ] Homebrew package manager: `$ brew cask install github-desktop`
```
Comment (iAmWillShepherd) [inline docs/process/testing.md]: If you remove the preceding `$`, people can copy and paste more easily into their terminals.

File: docs/process/testing.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
     - [ ] Windows: https://central.github.com/deployments/desktop/desktop/latest/win32
-      - [ ] Chocolatey package manager: c:\> choco install github-desktop
+      - [ ] Chocolatey package manager: `c:\> choco install github-desktop`
```
Comment (iAmWillShepherd) [inline docs/process/testing.md]: Same thing here with `c:\>`

File: docs/process/testing.md [hunk 3]
```diff
@@ -86,24 +86,25 @@
...
 	- [ ] 'Visibility can't be private error' shown, if user's plan is not sufficient
 	- [ ] A `.gitattributes` file is added to the repository as part of the initial commit
+	- [ ] If repo is forked and upstream remote does not match, then modal is surfaced with Ignore/Update button 
 
 ### Application
   - [ ] Minimize, Maximize, Close buttons work in top nav
       - [ ] If user zooms in and quits app, settings should be retained when reopened
   - [ ] Double-clicking local desktop icon opens the application (Mac only)
-  - [ ] Double clicking top nav bar toggles full-screen / last used screen-size (Mac only); Exit by (Mac: ^⌘f11; Windows: Alt)
+  - [ ] Double clicking top nav bar toggles full-screen / last used screen-size (Mac only); Exit by (Mac: `^⌘f11`; Windows: `Alt`)
```
Comment (iAmWillShepherd) [inline docs/process/testing.md]: Should this be `F11`?

File: docs/process/testing.md [hunk 4]
```diff
@@ -115,7 +116,7 @@
       - [ ] If checking for update or download latest update, the `Check for updates` button is disabled
       - [ ] Restarting the app automatically checks for updates
     - [ ] `Close` button closes modal 
-  - [ ] Preferences/Options (Mac/Windows: ⌘,)
+  - [ ] Preferences/Options (Mac/Windows: `⌘,`)
```
Comment (iAmWillShepherd) [inline docs/process/testing.md]: `Ctrl+,`

File: docs/process/testing.md (hunk 3 above)
Comment (tierninho) [reply docs/process/testing.md]: ^⌘f11 for mac, believe Alt for Windows

File: docs/process/testing.md (hunk 4 above)
Comment (tierninho) [reply docs/process/testing.md]: ✅

File: docs/process/testing.md (hunk 1 above)
Comment (tierninho) [reply docs/process/testing.md]: k

File: docs/process/testing.md (hunk 2 above)
Comment (tierninho) [reply docs/process/testing.md]: k

File: docs/process/testing.md (hunk 3 above)
Comment (lee-dohm) [reply docs/process/testing.md]: I prefer using `<kbd>` for key combinations. You may want to consider it for your documentation? It's kind of a cool trick: <kbd>^⌘f11</kbd>

**Comment by tierninho** [comment]
@iAmWillShepherd ready for round 2

File: docs/process/testing.md (hunk 3 above)
Comment (iAmWillShepherd) [reply docs/process/testing.md]: Holy 🐮, didn't know about that one.

Comment (tierninho) [reply docs/process/testing.md]: Nice, for next time! Thanks

**Pull request merged**